core/src/data/properties.cpp

```diff
@@ -31,14 +31,22 @@ void Properties::setSorted(std::vector<Item>&& _items) {
 const Value& Properties::get(const std::string& key) const {
     const static Value NOT_FOUND(none_type{});
 
-    const auto it = std::lower_bound(props.begin(), props.end(), key,
-                                     [](const auto& item, const auto& key) {
-                                         return keyComparator(item.key, key);
-                                     });
-
-    if (it == props.end() || it->key != key) {
+    const auto it = std::find_if(props.begin(), props.end(),
```

Comment:
I know and agree that these things compiler should figure out, just sharing what we found :D.

Yes it was with std::vector. Do not remember if it was release/debug build.